Greedy/test/GaleShapley.test.js

```diff
@@ -0,0 +1,36 @@
+import { expect, test } from 'vitest'
+import { stableMatching } from '../GaleShapley'
+
+test('Test Case 1', () => {
```

Comment:
A test case structure like this isn't helpful. Please see https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md#writing-good-tests (it's for the TS repo, but translates to JS just as well).

TL;DR: Use a `describe` block for Gale-Shapley and `it.each` for individual test cases if you have no descriptive labels for them.